crates/matrix-sdk-base/src/store/send_queue.rs

```diff
@@ -248,6 +260,13 @@ pub enum DependentQueuedRequestKind {
     },
 }
 
+/// If parent_is_thumbnail_upload is missing, we assume the request is for a
+/// file upload following a thumbnail upload.
```

Comment:
Hmm, is that because that was the case before the data changes here? Maybe worth mentioning in the comment if so.

Reply:
Yes, that's exactly it. Have amended the comment, good idea. 👍